modules/cockroachdb/cockroachdb.go

```diff
@@ -233,6 +239,38 @@ func addTLS(ctx context.Context, container testcontainers.Container, opts option
 	return nil
 }
 
+// runStatements runs the configured statements against the CockroachDB container.
+func runStatements(ctx context.Context, container testcontainers.Container, opts options) error {
+	if len(opts.Statements) == 0 {
+		return nil
+	}
+
+	port, err := container.MappedPort(ctx, defaultSQLPort)
+	if err != nil {
+		return fmt.Errorf("mapped port: %w", err)
+	}
+
+	host, err := container.Host(ctx)
+	if err != nil {
+		return fmt.Errorf("host: %w", err)
+	}
+
+	db, err := sql.Open("pgx/v5", connString(opts, host, port))
+	if err != nil {
+		return fmt.Errorf("sql.Open: %w", err)
+	}
+	defer db.Close()
```

Comment:
bug: lets ensure that the error from close is returned as otherwise we could miss an issue.